src/storage/s3.rs

```diff
@@ -17,14 +17,17 @@ pub(crate) use test::TestS3;
 
 pub(crate) static S3_BUCKET_NAME: &str = "rust-docs-rs";
 
-pub(crate) struct S3Backend<'a> {
+pub(crate) struct S3Backend {
     client: S3Client,
-    bucket: &'a str,
+    bucket: String,
```

Comment:
I'd rather make this `&'static str` instead; it communicates the intent more clearly and avoids a copy.

Reply:
It can't be `'static`, the bucket name is generated dynamically in `TestS3`.

Reply:
You could use a `Cow<'static, str>`

Reply:
This is initialized just a single time when the application starts, is that really worth?